sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -639,17 +640,21 @@ public ServiceBusSenderClient buildClient() {
 
     /**
      * Builder for creating {@link ServiceBusProcessorClient} to consume messages from a session-based Service Bus
-     * entity. {@link ServiceBusProcessorClient ServiceBusProcessorClients} provides a push-based mechanism that
-     * invokes the message processing callback when a message is received or the error handler when an error occurs
-     * when receiving messages. To create an instance, therefore, configuring the two callbacks -
-     * {@link #processMessage(Consumer)} and {@link #processError(Consumer)} are necessary. By default, a
-     * {@link ServiceBusProcessorClient} is configured with auto-completion and auto-lock renewal capabilities.
-     * {@link ServiceBusProcessorClient ServiceBusProcessorClients} created by this builder can be configured to
-     * consume messages from {@link #maxConcurrentSessions(int) multiple sessions concurrently}. By default, the
-     * processor will receive messages from a single session with rollover.
+     * entity. {@link ServiceBusProcessorClient} processes messages and errors via {@link #processMessage(Consumer)}
+     * and {@link #processError(Consumer)}. When the processor finishes processing a session, it tries to fetch the
+     * next session to process.
+     *
+     * <p>
+     * By default, the processor:
+     * <ul>
+     *     <li>Automatically settles messages. Disabled via {@link #disableAutoComplete()}</li>
+     *     <li>Processes 1 session concurrently. Configured via {@link #maxConcurrentSessions(int)}</li>
+     *     <li>Invokes 1 instance of {@link #processMessage(Consumer) processMessage consumer}. Configured via
```

Comment:
Should we explain about lock renewal also here ?

Reply:
We don't have that ability in the session processor? I don't think we have auto renew session lock.